src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -403,24 +400,38 @@ void GcsServer::StoreGcsServerAddressInRedis() {
   RAY_LOG(INFO) << "Finished setting gcs server address: " << address;
 }
 
-void GcsServer::InitResourceReportPolling(const GcsInitData &gcs_init_data) {
-  gcs_resource_report_poller_.reset(new GcsResourceReportPoller(
-      raylet_client_pool_, [this](const rpc::ResourcesData &report) {
-        gcs_resource_manager_->UpdateFromResourceReport(report);
-      }));
+void GcsServer::InitRaySync(const GcsInitData &gcs_init_data) {
+  /*
+    The current synchronization flow is:
```

Comment:
@rkooo567  Btw, right now poller is not put in ray sync because of resource update. I'll put it there once I delete the legacy code.